deployment/terraform/metrics.go

```diff
@@ -80,6 +82,27 @@ func (t *Terraform) UploadDashboard(dashboard string) (string, error) {
 	return url, nil
 }
 
+type PanelData struct {
+	Id        int
+	Title     string
+	Legend    string
+	Height    int
+	Width     int
+	PosX      int
+	PosY      int
+	Query     string
+	Threshold float64
+}
+
+const (
+	panelWidth  = 12
+	panelHeight = 9
```

Comment:
Maybe some comments to explain how you got these numbers?

Reply:
Sure!